test/integration/08-specter.spec.ts

```diff
@@ -62,38 +62,39 @@ it("createWallet", async () => {
   // expect(balance).toBe(0)
 })
 
-it("deposit to bitcoind", async () => {
```

Comment:
This test is being commented out (at least for now). Many of SpecterWallet methods are tightly coupled to the outside (default) wallet being bypassed in tests. In addition to also considering lnd as the onchain wallet.